Javascript/Ez Namey /namey.js

```diff
@@ -0,0 +1,43 @@
+let firstWords = [
+  "Amazing",
+  "Creative",
+  "Super",
+  "Fantastic",
+  "Smart",
+  "Future",
+  "Ultimate",
+  "Generation",
+  "Brilliant",
+  "Innovative",
+];
+let secondWords = [
+  "Solutions",
+  "Ideas",
+  "Concepts",
+  "Technologies",
+  "Systems",
+  "Creations",
+  "Inventions",
+  "Platforms",
+  "Ventures",
+  "Projects",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
+
+let startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
```

Comment:
Using the same `randomNumber` for both `firstWords` and `secondWords` will limit the amount of combinations you can get generated.  In a way you will always get "Amazing Solutions", "Creative Ideas" etc, but would never get "Amazing Ideas", or "Innovative Solutions".

Reply:
***Got it. Changed.***